setup.cfg

```diff
@@ -127,6 +127,9 @@ filterwarnings =
     # is being ignored
     #
     #
+    # See https://github.com/pandas-dev/pandas/issues/26367; this can be
+    # removed when our minimum pandas version is >=0.25
```

Comment:
why isn't it already? wasn't that released like  a million years ago now?

Reply:
Should we deal with our min versions for 2.1? If we haven't already?

Reply:
Pandas 0.23 doesn't seem compatible with python 3.9 anyway, so I will bump in this PR.